app/src/main/java/com/lagradost/cloudstream3/MainActivity.kt

```diff
@@ -607,6 +615,10 @@ class MainActivity : AppCompatActivity(), ColorPickerDialogListener {
             initAll()
             // No duplicates (which can happen by registerMainAPI)
             apis = allProviders.distinctBy { it }
+
+            if (githubApi.getLatestLoginData() != null && settingsManager.getBoolean(getString(R.string.automatic_cloud_backups), false)){
```

Comment:
This is also a race condition as initAll() inits the providers, not apis
Move this check to after `inAppAuths.amap`